ServerCore/Pages/Teams/MergeInto.cshtml

```diff
@@ -0,0 +1,53 @@
+﻿@page "/{eventId}/{eventRole}/Teams/{teamId}/MergeInto"
+@model ServerCore.Pages.Teams.MergeIntoModel
+
+@{
+    ViewData["Title"] = "Merge Team Into Another";
+    //Needs route data - ViewData["AdminRoute"] = "/Teams/Delete";
+    //Needs route data - ViewData["AuthorRoute"] = "/Teams/Status";
+    //Needs route data - ViewData["PlayRoute"] = "/Teams/Delete";
+    Layout = "_teamLayout.cshtml";
+}
+
+<h2>@Html.DisplayFor(model => model.Team.Name): Merge Into Another Team</h2>
+<div>
+    <a asp-page="./Details" asp-route-teamId="@Model.Team.ID">Cancel</a>
+</div>
+
+<h3>Are you sure you want to merge this team into another team?</h3>
+<div>
+    <hr />
+    <form method="post">
+        <dl>
+            <!-- class="dl-horizontal"> -->
+            <dt>
+                Team name
+            </dt>
+            <dd>
+                @Html.DisplayFor(model => model.Team.Name)
+            </dd>
+            <dt>
+                Primary contact e-mail
+            </dt>
+            <dd>
+                @if (Model.Team.PrimaryContactEmail == null)
+            {
+                <div>(none)</div>
+            }
+            else
+            {
+                @Html.DisplayFor(model => model.Team.PrimaryContactEmail)
+            }
+            </dd>
+            <dt>
+                Merge Into
```

Comment:
Maybe a disclaimer to the admin using this page that an email will be sent to team members?